### PR TITLE
Update/Cleanup `Mappers' Guild` and `Featured artist showcase beatmaps`

### PR DESCRIPTION
**Pull request by Nivalyx**
Follow-up of [#6052](https://github.com/ppy/osu-wiki/pull/6052) :

- Add `Showcase beatmaps` sub-section to the `Mappers' Guild` article
- Add related images to the `Mappers' Guild` article
- Update the `Featured artist showcase beatmaps` article to include showcase beatmaps from August
- ...and other sorts of general cleanup to both articles

### CODE REVIEW

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Featured artist showcase beatmaps
 
-Members of the [Mappers' Guild](/wiki/Mappers_Guild) occasionally create exclusive "showcase beatmaps" by using songs and other materials from osu!'s unreleased [featured artists](/wiki/Featured_Artists). When a featured artist is publicly announced, these beatmaps are covered in the corresponding [newsposts](https://osu.ppy.sh/home/news) and are often promoted on the [@osugame Twitter](https://twitter.com/osugame).
+Members of the [Mappers' Guild](/wiki/Mappers_Guild) may occasionally create exclusive "showcase beatmaps" by using songs and other materials from osu!'s unreleased [featured artists](/wiki/Featured_Artists). Upon the announcement of the corresponding featured artist, these beatmaps are highlighted in the [newspost](https://osu.ppy.sh/home/news) and are often promoted on the [@osugame Twitter](https://twitter.com/osugame "Twitter") as well.
```
Comment (TicClick) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:3]: we've decided to throw the alt text away: #6060
```suggestion
Members of the [Mappers' Guild](/wiki/Mappers_Guild) may occasionally create exclusive "showcase beatmaps" by using songs and other materials from osu!'s unreleased [featured artists](/wiki/Featured_Artists). Upon the announcement of the corresponding featured artist, these beatmaps are highlighted in the [newspost](https://osu.ppy.sh/home/news) and are often promoted on the [@osugame Twitter](https://twitter.com/osugame) as well.
```

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 2]
```diff
@@ -387,12 +387,18 @@ This article lists all the showcase beatmaps that have been created by the membe
...
+  - (![][osu!]) [Release Hallucination - Ariadne](https://osu.ppy.sh/beatmapsets/1550194) hosted by ![][flag_PL] [Zelq](https://osu.ppy.sh/users/8953955)
 
 ## History
 
-Even from its very beginning, the Mappers' Guild has been set into pace with "creating promotional beatmap packs for upcoming featured artists" <!-- https://osu.ppy.sh/community/forums/topics/875743 --> as one of its primary goals; in fact, the six beatmaps that were made for the [MOtOLOiD featured artist announcement](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) were also the Guild's first ever creations. 
+From its very beginning, the Mappers' Guild has been set into pace with "creating promotional beatmap packs for upcoming featured artists" <!-- https://osu.ppy.sh/community/forums/topics/875743 --> as one of its primary goals. This was reflected in the Guild's first ever major project, in where members of the Guild had prepared and ranked six beatmaps of [licensed MOtOLOiD songs](https://osu.ppy.sh/beatmaps/artists/19) long prior to [the corresponding artist's announcement](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid).
```
Comment (TicClick) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:399]:
```suggestion
From its very beginning, the Mappers' Guild has been set into pace with "creating promotional beatmap packs for upcoming featured artists"<!-- https://osu.ppy.sh/community/forums/topics/875743 --> as one of its primary goals. This was reflected in the Guild's first ever major project, in where members of the Guild had prepared and ranked six beatmaps of [licensed MOtOLOiD songs](https://osu.ppy.sh/beatmaps/artists/19) long prior to the [corresponding artist's announcement](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid).
```

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 3]
```diff
@@ -387,12 +387,18 @@ This article lists all the showcase beatmaps that have been created by the membe
...
 
-As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were often restricted to only a handful of mappers who were predominantly of the [osu!](/wiki/Game_mode/osu!) game mode; however, the Guild's [shift to an open and automated entry system in early 2019](/wiki/Mapping_and_Modding_Timeline#march.5) gradually allowed more and more creators to participate in the creation of showcase beatmaps for future featured artists over time regardless of their game mode.
+As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were often restricted only to a handful of mappers who were predominantly of the [osu!](/wiki/Game_mode/osu!) game mode]. However, as the Guild [began to shift to an open and automated entry system in early 2019](/wiki/Mapping_and_Modding_Timeline#march.5), more and more mappers were drawn into the creation of showcase beatmaps for future featured artists over time regardless of their game mode.
```
Comment (TicClick) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:401]: long links are poor style
```suggestion
As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were often restricted only to a handful of mappers who were predominantly of the [osu!](/wiki/Game_mode/osu!) game mode]. However, as the Guild began to shift to an open and automated entry system [in early 2019](/wiki/Mapping_and_Modding_Timeline#march.5), more and more mappers were drawn into the creation of showcase beatmaps for future featured artists over time regardless of their game mode.
```

File: wiki/Mappers_Guild/en.md [hunk 4]
```diff
@@ -44,9 +48,15 @@ Points are earned for different mapping-related tasks. Earnings vary between tas
 
 Difficulty creation tasks earn a user more points if the beatmap is associated with a quest. Completing a quest before its deadline provides users with bonus points for each difficulty mapped and a flat bonus for all users involved in the quest. This flat bonus is lost after the deadline is passed, however difficulty bonuses are kept with diminishing returns after 0, 20 and 40 day increments beyond the deadline.
 
+### Showcase beatmaps
+
+*Main article: [Featured artist showcase beatmaps](wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps)*
+
+Members of the Mappers' Guild may also participate in the creation of showcase beatmaps for future featured artist announcements. Upon the announcement of the corresponding featured artist, these beatmaps are highlighted in the [newspost](https://osu.ppy.sh/home/news) and are often promoted on the [@osugame Twitter](https://twitter.com/osugame "Twitter") as well.
```
Comment (TicClick) [inline wiki/Mappers_Guild/en.md:55]: please remove alt text here as well
also, newspost -> news? sounds more fitting
```suggestion
Members of the Mappers' Guild may also participate in the creation of showcase beatmaps for future featured artist announcements. Upon the announcement of the corresponding featured artist, these beatmaps are highlighted in the [newspost](https://osu.ppy.sh/home/news) and are often promoted on the [@osugame Twitter](https://twitter.com/osugame) as well.
```

**Review by TicClick — CHANGES_REQUESTED** [review]
minor corrections, the texts read alright to my taste anyway

**Comment by Nivalyx** [comment]
ah, ok, applied everything @TicClick 👍 

for that one suggestion i think both `news` and `news post` work just fine tho, so i decided to keep it as is. feel free to change it if you feel necessary tho :P

**Review by TicClick — APPROVED** [review]
nah it's minor

**Pull request merged**